addons/l10n_in_edi/views/account_move_views.xml

```diff
@@ -7,9 +7,16 @@
         <field name="arch" type="xml">
             <xpath expr="//group[@name='sale_info_group']" position="inside">
                 <field name="l10n_in_edi_show_cancel" invisible="1"/>
+                <field name="l10n_in_edi_show_negative_line_info" invisible="1"/>
                 <field name="l10n_in_edi_cancel_reason" invisible="country_code != 'IN' or state != 'posted' or not l10n_in_edi_show_cancel"/>
                 <field name="l10n_in_edi_cancel_remarks" invisible="country_code != 'IN' or state != 'posted' or not l10n_in_edi_show_cancel"/>
             </xpath>
+            <xpath expr="//header" position="after">
+                <div invisible="not l10n_in_edi_show_negative_line_info" class="alert alert-info mb-0" role="alert">
```

Comment:
it can be annoying if it is always there.

Reply:
Then we can add button like not show again!
what you think?